linode/manager
# fix: [M3-6627] - Fix `cy.defer()` TypeScript errors

## Description 📝
This fixes the `cy.defer()` custom Cypress command signature and implementation to resolve TypeScript errors. The new implementation is a pretty thin wrapper around `cy.wrap()`, which accomplishes the same thing but isn't as ergonomic.

## How to test 🧪
1. Confirm that the tests continue to pass via the automated Cypress run
2. Check out this PR, and confirm there aren't any type errors in `cypress/support/commands.ts` at or around line 38.
